CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.0.0 (October 11, 2016)
+    - Added Monitoring enabled option
+    - Added submodule support
+    - Added VPC support
+    - Added instance IAM role /profile support
+    - Removed CPU checks from tiopatinhas and added weight factor (now you decide the ratio of machines that tiopatinhas will handle and the ASG handles everything else)
+    - Fixes on Market crashed handlers and LB bugs
+
```

Comment:
The use of `marked-for-termination` status to set an instance as not live might be worth of mentioning here?